decentraland/unity-renderer
# chore: Add fade in animation for asset bundles

## What does this PR change?

Add the custom fade-in effect for when assets load

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=chore/asset-bundle-fade-in-animation
2. Walk around and see that the custom fade in effect is present

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
